models/banner.js

```diff
@@ -0,0 +1,139 @@
+'use strict';
+
+const Joi = require('joi');
+const mutate = require('../lib/mutate');
+
+const MODEL = {
+    id: Joi
+        .number()
+        .integer()
+        .positive(),
+
+    message: Joi
+        .string()
+        .max(512)
+        .description('Body of banner to display')
+        .example('Due to planned upgrade of Kubernetes, Screwdriver will be down'),
+
+    isActive: Joi
+        .boolean()
+        .description('Flag if the banner is active')
+        .example(true)
+        .default(false),
```

Comment:
Just as a reminder, setting default value here is not going to automatically generate the value for this field in DB. If we really want this to be default value in the DB, we still need to set it in model create part. This will generate the value only when you do joi.validate with the schema, but we're not using the joi object coming from joi.validate when we save it...